SpringVoting/src/voting/controllers/AjaxController.java

```diff
@@ -0,0 +1,71 @@
+package voting.controllers;
+
+import java.util.List;
+
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import voting.dao.QuestionDAO;
+import voting.dao.UserDAO;
+import voting.model.ActiveVotings;
+import voting.model.SignInUser;
+
+@Controller
+@RequestMapping("/ajax")
+public class AjaxController {
+	
+	@Autowired
+	private UserDAO userDAO;
+	@Autowired 
+	private QuestionDAO questionDAO;
+	
+	@RequestMapping(value="/checkUniqueEmail")
+	public void checkUniqueEmail(String email, HttpServletResponse response){

```

Comment:
is it nesseccary to specify @RequestMapping annotation if you use the same name as method name?